src/main/java/org/sbml/spatial/segmentation/UnetTrainAndSave.java

```diff
@@ -67,20 +25,9 @@ public static void main(String[] args) {
 			String directory = System.getProperty("user.dir");
 			String dataPath = directory + File.separator + "dataset";
 
-			DataNormalization scaler = new ImagePreProcessingScaler(); // scale image between 0 and 1
-			UnetPathLabelGenerator labeler = new UnetPathLabelGenerator();
-
 			File rootDir = new File(dataPath + File.separator + "small_dataset");
-			String[] allowedExtensions = BaseImageLoader.ALLOWED_FORMATS;
-			Random rng = new Random();
-			FileSplit inputSplit = new FileSplit(rootDir, allowedExtensions, rng);
-			ImageRecordReader imageRecordReader = new ImageRecordReader(HEIGHT, WIDTH, CHANNELS, labeler);
-			imageRecordReader.initialize(inputSplit);
-			int labelIndex = 1;
-			DataSetIterator imageDataSetIterator = new RecordReaderDataSetIterator(imageRecordReader, batchSize,
-					labelIndex, labelIndex, true);
-			scaler.fit(imageDataSetIterator);
-			imageDataSetIterator.setPreProcessor(scaler);
+			PreProcess prep = new PreProcess(rootDir, batchSize);
+			DataSetIterator imageDataSetIterator = prep.dataProcessed();
 
 			Map<Integer, Double> learningScheduleMap = new HashMap<>();
 			learningScheduleMap.put(0, 0.00005);
```

Comment:
Is this map ever used?

Reply:
When we use line no. 42 to initialize ComputationGraph model, the map is used. I will comment it out.